release_notes/ocp_3_7_release_notes.adoc

```diff
@@ -1839,11 +1839,22 @@ created with lowercase letters.
 later) the default values for `clusterNetworkCIDR` and `hostSubnetLength`
 changed. If the inventory file did not specify corresponding inventory
 variables, the upgrade would fail. This caused the controller service to not
-start back up. This bug fix requires that the inventory variables be set before
-upgrading or installing. As a result, if the required inventory variables are
-not set, the upgrade or installation will stop and tell the administrator to set
-the variables.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1451023[*BZ#1451023*])
+start back up. The following Ansible variables must now be set in your inventory
+file before upgrading or installing:
```

Comment:
I do not think this is needed for a fresh install, so here "or installing" should be removed.